parlai/core/build_data.py

```diff
@@ -244,9 +246,9 @@ def modelzoo_path(datapath, path):
     """
     if path is None:
         return None
-    if not path.startswith('models:'):
+    if not path.startswith('models:') and not path.startswith('internal:'):
```

Comment:
What do you think about `internalzoo:`?

I personally think `models:` is obtuse and it might make sense for us to "switch" to `zoo:`.